src/scss/includes/direction.scss

```diff
@@ -9,6 +35,10 @@
   }
 }
 
+.panel-content {
```

Comment:
This is applied on the global scope. If that's on purpose, it should be defined in the panel stylesheet. Otherwise, it should be scoped with a parent selector.